CONTRIBUTING.md

```diff
@@ -2,25 +2,44 @@
 
-Please also focus attention on the requirements themselves and **not** on the surrounding text or on the introductory chapters (files 0x01-0x04). This because the text might have become outdated where requirements have changed and also because our goal is to significantly cut down this text anyway in 5.0.
+We will try, on a best efforts basis, to address issues and push changes to the "bleeding edge" master branch within the 5.0 folder **without altering current numbering** and using tagging ([see below](CONTRIBUTING.md#use-tags-to-describe-the-change)) to make changes clearer. This is in order to make it easier to use this branch on an ongoing basis. 
+
+We will no longer be accepting changes to the 4.0 folder which is now fixed at the 4.0.3 release.
 
-### Where do I start?
+Please also focus attention on the requirements themselves and **not** on the surrounding text or on the introductory chapters (files 0x01-0x04). This is because the text might have become outdated where requirements have changed and also because our goal is to significantly cut down this text in version 5.0.
 
-The most important thing you can do at the moment is give your feedback on open issues which we are currently discussing.
+## How can I help?
+
+Reading through the most recent version of ASVS is a great place to start.
+
+After familiarizing yourself with the current version, the next area to focus on is the Issues section. 
 
 There are two key types right now:
 
 * [![GitHub issue custom search](https://img.shields.io/github/issues-search?query=repo%3Aowasp%2Fasvs%20is%3Aopen%20is%3Aissue%20label%3A%22Community%20wanted%22&style=flat&label=Community%20Wanted&labelColor=%23BFD4F2&color=grey)](https://github.com/OWASP/ASVS/issues?q=is%3Aopen+is%3Aissue+label%3A%22Community+wanted%22) - These are issues where we would really benefit from more eyes on a particular issue.
 * [![GitHub issue custom search](https://img.shields.io/github/issues-search?query=repo%3Aowasp%2Fasvs%20is%3Aopen%20is%3Aissue%20label%3A%22Community%20needed%22&style=flat&label=Community%20Needed&labelColor=%23fbca04&color=grey)](https://github.com/OWASP/ASVS/issues?q=is%3Aopen+is%3Aissue+label%3A%22Community+needed%22) - These are issues where the relevant items will not get progressed without community input.
 
+
+Please log ideas, issues or questions here: https://github.com/OWASP/ASVS/issues. It’s helpful to share if you have any ideas or if you find any bugs. We may subsequently ask you to open a pull request, https://github.com/OWASP/ASVS/pulls, based on the discussion in the issue. 
```

Comment:
Cleanup "Please log your ideas, issues, or questions here: https://github.com/OWASP/ASVS/issues. Sharing your ideas or reporting bugs is helpful. Based on the discussion, we might ask you to open a pull request here: https://github.com/OWASP/ASVS/pulls."

Reply:
Done